server/handlers/meshery_filter_handler.go

```diff
@@ -130,6 +131,10 @@ func (h *Handler) handleFilterPOST(
 
 		mesheryFilter := parsedBody.FilterData
 
+		// encode the filter binary data to base64 string before processing it further.
+		filterData := base64.StdEncoding.EncodeToString([]byte(mesheryFilter.FilterFile))
+		mesheryFilter.FilterFile = filterData
```

Comment:
```suggestion
		bytFilter := []byte(mesheryFilter.FilterFile)
		mesheryFilter.FilterFile = bytFilter
```

Reply:
Although, this would require couple more changes and normalization of Filters struct https://github.com/meshery/meshery/blob/master/server/models/meshery_filter.go#L16 with the one used by Remote Provider. 

As FilterFile should be of type `bytes` according to the struct in Remote Provider but in Meshery Server its defined as of type `string`

Reply:
We use similar strategy for storing Application source content on upload.
https://github.com/meshery/meshery/blob/master/server/handlers/meshery_application_handler.go#L159

Reply:
Ok

Reply:
Thanks Pranav 🙌